Can't compile fixed sample

Following sample
```csharp
class C
{
    static S s = new S();

    public void M()
    {
        fixed (int* ptr = s.myFixedField)
        {
            int p = ptr[5];
        }
    }
}
```
Shows
error CS0213: You cannot use the fixed statement to take the address of an already fixed expression

While this one works fine
```csharp
class C
{
    static S s = new S();

    public void M()
    {
        int* ptr = s.myFixedField;
        int p = ptr[5];
    }
}
```


---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5c0320fd-c7ac-200e-7e50-6b3ddcaa6a52
* Version Independent ID: 6ea1af7f-2251-f05f-4834-5a00e57fc5fb
* Content: [What's new in C# 7.3](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7-3)
* Content Source: [docs/csharp/whats-new/csharp-7-3.md](https://github.com/dotnet/docs/blob/master/docs/csharp/whats-new/csharp-7-3.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/docs/csharp/whats-new/csharp-7-3.md
+++ b/docs/csharp/whats-new/csharp-7-3.md
@@ -66,7 +66,7 @@ class C
 {
     static S s = new S();
 
-    public void M()
+    unsafe public void M()
     {
         fixed (int* ptr = s.myFixedField)
         {

